.github/workflows/vulnerability-triage.yml

```
@@ -9,6 +9,7 @@ permissions:
   contents: read
   packages: read
   security-events: read
+  vulnerability-alerts: read
```

actionlint (1.7.12)

[error] unknown permission scope "vulnerability-alerts". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)